main

```diff
@@ -3,6 +3,10 @@
 Module for launching formater
 """
 from cli import init_argparser
+from db import init_styles_db
+
 
 if __name__ == '__main__':
-    init_argparser()
\ No newline at end of file
+    init_styles_db()
+    init_argparser()
```

Comment:
It is more logical to call `init_argparser` before DB, because you may want to change DB location from options.